cmd/datool/datool.go

```diff
@@ -92,6 +92,7 @@ type ClientStoreConfig struct {
 	SigningWallet         string        `koanf:"signing-wallet"`
 	SigningWalletPassword string        `koanf:"signing-wallet-password"`
 	MaxStoreChunkBodySize int           `koanf:"max-store-chunk-body-size"`
+	DisableChunkedStore   bool          `koanf:"disable-chunked-store"`
```

Comment:
I believe there is a movement to avoid boolean flags with a negative assertion. I would rename it to `EnableChunkedStore` and set the default to true.

Reply:
makes sense! thanks